### DISCUSSION TITLE
Problem with installation

### DISCUSSION THREAD

**Discussion by ghost** [toshiakit/MatGPT#14] [Q&A] [answered]
Hi, when I want to install the app I get this:
![imagen](https://github.com/toshiakit/MatGPT/assets/37338678/301fad10-7c51-470e-9183-81fd30e6ee91)

**Comment by toshiakit** [comment, marked as answer]
What version of MATLAB do you have? chatGPT class and child class chatter use Name=Value syntax in function calls, introduced in R2021a.

**Reply by ghost** [reply]
It's 2020a ...
![imagen](https://github.com/toshiakit/MatGPT/assets/37338678/91334ba8-9b84-45d9-9e09-ab24e0a8a4c1)

Thank you very much

**Reply by toshiakit** [reply]
You can modify the code to use older syntax, but it would be easier if you can update your version to R2021a or later.

**Reply by ghost** [reply]
ok, thanks!
